### ISSUE TITLE
Add View to Clip Leaves Artifact

### ISSUE THREAD

**Issue by WandererFan** [WandererFan/FreeCAD-TechDraw#49]
Add a View to a Clip group leaves a visual artifact at the Views old location.  Need to perform a QGIV->hide() before adding to Clip group.   Command does not have direct access to QGIV, nor does ViewProvider. 

```
App::Document* doc = view->getDocument();
Gui::Document* activeGui = Gui::Application::Instance->getDocument(doc);
Gui::ViewProvider* vp = activeGui->getViewProvider(view);
vp->hide();   //doesn't help! need to tell QGIV to hide
```

**Comment by WandererFan** [comment]
fix to #10 adds ability to toggle Visibility property, but need a way to redraw page in Command ClipAdd,
after Visibility = False and Clip.AddView().
document recompute doesn't help. 

Artifact disappears if Page is Zoomed in&out.

**Comment by WandererFan** [comment]
fixed!!  master PR#228
